queue_test.go

```diff
@@ -133,30 +150,30 @@ func (q *QueueTestSuite) TestQueue_UnparseableBody() {
 	q.assert.Contains(q.sqsClient.Deleted, badMessage)
```

Comment:
You may want to assert that the q length is only 2. Right now if the unparseable worker is enqueued, the test would not fail.